src/raml-cop.js

```diff
@@ -38,7 +39,7 @@ const validate = function (filename, options) {
 
       // Generic error
       if (!err.parserErrors) {
-        err.results = [{ src: filename, message: err.message }];
+        err.results = [{ src: filename, message: err.message, isWarning: err.isWarning }];
```

Comment:
I don't think this is necessary. I think the `isWarning` property only applies to parser errors not to generic errors.